### ISSUE TITLE
potential error for non-ref call imputation 

### ISSUE THREAD

**Issue by yangjl** [yangjl/imputeR#6]
focal parent's genotype:
    0     1     2     3 
10946  1098  1674 10281 

after imputation, we got:
    0     1 
16298  7701 

It does not quite make sense.
try to fix it.

**Comment by rossibarra** [comment]
This may be hard to fix. First remember that under hwe we expect something
like this. Let's say allele is at 4%. Then we expect 0.016 homozygous
genotypes (0 in our sample of 70 ). But we expect ~7% of individuals to be
heterozygous. 40% (half of the 80% error) of those hets will be called as
homozygous, giving us an observed count of 2 homozygous parents. So if we
impute correctly for rare alleles most the homozygous should disappear. I
would look at this error rate for higher freq alleles (say 0.2 or so) where
we expect under HWE to see some real homozygous.

**Comment by yangjl** [comment]
We are calculating posterior `p(G | theta)`   instead of likelihood `p(theta | G)` currently. 
HW `p(G)` is the prior in this case. With this method, when the data is limited, we will put two much weight on prior. This becomes problematic, especially for rare alleles. For example, a rare site with no kids data, we tend to impute it as 0 because of HW. Or if we observed some 2 in the kids, we will tend to impute it as 1.
To fix it, I simply changed to calculate likelihood and remove HW part `p(G)`.
